Source/Krypton Components/Krypton.Navigator/Palette/PaletteRedirectRibbonTabContent.cs

```diff
@@ -45,9 +45,9 @@ public class PaletteRedirectRibbonTabContent : PaletteRedirect
         /// <param name="target">Initial palette target for redirection.</param>
         public PaletteRedirectRibbonTabContent(PaletteBase target)
             : this(target,
-                   null, null, null, null, null, null,
-                   null, null, null, null, null, null,
-                   null, null, null, null, null, null)
+                   null!, null!, null!, null!, null!, null!,
```

Comment:
These are clearly null, so the base has to take them. So make the base params nullable